charts/aspenmesh-collector/Chart.yaml

```diff
@@ -1,15 +1,15 @@
 apiVersion: v2
 name: aspenmesh-collector
 type: application
-version: 0.1.1
+version: 0.1.2
 appVersion: 0.1.0
 description: Aspen Mesh Collector
 icon: https://aspenmesh.github.io/aspenmesh-charts/docs/images/aspenmesh.png
 dependencies:
   - name: cert-manager
+    #condition: cert-manager.enabled
```

Comment:
we can remove this comment

Reply:
> I assume that by simply including the tarball, we'll get that installed for free by running the same `helm install` command?

That's correct. Had the helm chart been publicly accessible, it would have been included as a dependency as the other charts.

Reply:
Removed the comment.